dashboard/src/actions/comparisonActions.js

```diff
@@ -111,11 +158,9 @@ export const parseChartData = () => (dispatch, getState) => {
     }, Object.create(null));
 
     for (const [key, value] of Object.entries(result)) {
-      console.log(key);
-
       const map = {};
```

Comment:
Noting (new) line 158, should we be using `Object.create(null)` at line 161 instead of `{}`?